microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 
+
+## Background
+The WebView2 team tracks data to help understand and prioritize our customers' requirements, diagnose and debug issues, and overall improve WebView2 as a product.
+Developers consent to this data collection in the WebView2 Runtime Terms and Conditions License, or in the Windows license. Additionally, we follow the standards outlined in the [Microsoft Edge Privacy Whitepaper](https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/).
+
+In general, all of our data collection goes through the same strict standards as Microsoft Edge. The main telemetry sources are Chromium and Edge telemetry infrastructure, Windows data reporting, and Watson (Microsoft Internal) infrastructure to collect crash dumps.
+With any data the WebView2 team collects, we comply/conform with the laws and regulations about best privacy practices, and have guardrails to protect against privacy issues.
+
+## Optional Diagnostic Data
+
+WebView2 and Edge respect the Windows Optional Diagnostic Data toggle, so end users have control over any optional data WebView2 collects. This toggle can only be modified by the end user. The only mandatory data that is collected is the [required events](https://learn.microsoft.com/en-us/windows/privacy/required-windows-11-diagnostic-events-and-fields) which are required from any app running on Windows.

```

Comment:
```suggestion
WebView2 and Microsoft Edge respect the Windows Optional Diagnostic Data toggle, so end users have control over any optional data WebView2 collects. This toggle can only be modified by the end user. The only mandatory data that is collected is the [required events](https://learn.microsoft.com/en-us/windows/privacy/required-windows-11-diagnostic-events-and-fields) which are required from any app running on Windows.
```
i've seen differing guidelines but the policy for this repo is never say Edge without Microsoft.

Reply:
Resolving this comment as this paragraph is going to be removed. @mikehoffms do you know the filepath to get to this doc - https://support.microsoft.com/en-us/windows/diagnostics-feedback-and-privacy-in-windows-28808a2b-a31b-dd73-dcd3-4559a5199319.